Automattic/pocket-casts-ios
# End of Year: show longest episode played

| 📘 Project: #376 |
|:---:|

Show the longest episode played story.

<img src="https://user-images.githubusercontent.com/7040243/196802873-28834c88-1f2f-4b9b-a0fd-1eb0d546c8f1.png" width="300">

## To test

1. Enable the `endOfYear` flag in `FeatureFlag.swift`
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. Skip to the latest story (the 7th)
5. ✅ Check that the longest episode that was played is shown, alongside its duration and podcast thumb

## Checklist
